README.md

```diff
@@ -133,19 +133,21 @@ val certs: List<Certificate> = response.map { Certificate.deserialize(it) }
 
 // Check if app has the root cert of Certificate Authority (CA). 
 
-// Validate cert matches the certificate request (so generated key) 
+// Validate cert matches the certificate request (so generated key)
+val result: Result<Unit> = TrifleApi.verifyAttributes(certs[0], certReq)
+
 // and the root (so it has been generated by the right CA).
-val result: Result<Unit> = TrifleApi.verify(certs[0], certReq, certs, root)
+val result: Result<Unit> = TrifleApi.verifyChain(certs, root)
 
 // Once it passes validation, certReq is no longer needed, and it can be deleted
 // Store cert along with the respective keyHandle
         
 // To check only for the validity of a stored cert, you can do either of below choices
 // Option 1 is a more complete check of the device cert and the full chain
-val result: Result<Unit> = TrifleApi.verify(cert, ancestorCertificateChain = certs.drop(1))
+val result: Result<Unit> = TrifleApi.verifyChain(certs)
 
 // option 2 only checks the validity of the device cert
-val result: Result<Unit> = TrifleApi.verify(cert)
+val result: Result<Unit> = TrifleApi.verifyValidity(cert)
```

Comment:
does the client need to do all these checks? why do we expose all these certs?
sorry if these questions a little dumb, but im a security noob:p

why can't we just have `TrifleApi.verify(keyHandle, certs)`
and it does all these for us?

Reply:
Yes the client should perform checks on these. We don't need keyhandle as a parameter. 2/3 of these have optional parameters. If we were to consolidate all 3 we'd be back to where we were in PR #156.

Reply:
Im good with dividing up verify to have more meaningful APIs. 
Each verify*() is intended to verify a common set of functionality as well as their own additional checks. 

Resolving this comment - @jamesalee213  happy to discuss more in sync.